specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,1918 @@
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Web/connectionGateways": {
+            "get": {
+                "tags": [
+                    "Connection Gateways"
+                ],
+                "summary": "List all of the connection gateways",
+                "operationId": "ConnectionGateways_ListAll",
+                "description": "Gets a list of gateways under a subscription",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/subscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The gateway definitions",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/ConnectionGatewayDefinition"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/connectionGateways": {
+            "get": {
+                "tags": [
+                    "Connection Gateways"
+                ],
+                "summary": "List all of the connection gateways",
+                "operationId": "ConnectionGateways_List",
+                "description": "Gets a list of gateways under a subscription and in a specific resource group",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/subscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/resourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/apiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The gateway definitions",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/ConnectionGatewayDefinition"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/connectionGateways/{connectionGatewayName}": {
+            "get": {
+                "tags": [
+                    "Connection Gateways"
+                ],
+                "summary": "Gets a specific gateway",
+                "operationId": "ConnectionGateways_Get",
+                "description": "Gets a specific gateway for under a subscription and in a specific resource group",
```

Comment:
"for under" is that proper English? above was just "under"

Reply:
Nope, that is not proper English. Fixed